examples/with-apollo/index.js

```diff
@@ -0,0 +1,52 @@
+const polka = require('polka')
+const bodyParser = require('body-parser')
```

Comment:
Let's import it as `const { json } = require('body-parser')` then update the usage below too